### ISSUE TITLE
LCP: allow Share / Open With for licenses from Files and email attachments

### ISSUE THREAD

**Issue by llemeurfr** [readium/r2-testapp-swift#309]
This was possible before and even if we can open a license from the app itself, it makes sense to import a license from Files and email attachments.

Tested on iOS13, on an iPhone 7.

**Comment by mickael-menu-mantano** [comment]
It was working last time I checked, but you might have faced the issue I mentioned here: https://github.com/readium/r2-testapp-swift/issues/263#issuecomment-544937685

"On a side note, after upgrading to iOS 13 to check if the issue was fixed in Files, Readium was not recognized for LCPL files anymore from the Files app. Adding a new Document Type in the app with exactly the same parameters but a different extension worked as intended. I can't explain it and didn't find a way to reset the shared Document Types so far."

It might be worth checking on another device that never had Readium installed before, or after reinstalling iOS.
